src/main/java/dtb/lego/truck/rest/component/events/entity/Components.java

```diff
@@ -0,0 +1,11 @@
+package dtb.lego.truck.rest.component.events.entity;
+
+/**
+ * Used to define the types of Components available.
+ */
+public enum Components {
```

Comment:
Enum are usually singular, in this case we should use Component

Reply:
Fixed, thanks